src/renderer/src/views/x/XFinishedRunningJobsPage.vue

```diff
@@ -197,6 +211,74 @@ onMounted(async () => {
             </div>
         </div>
 
+        <div v-if="jobsType == 'migrateBluesky'" class="container mt-3">
+            <div class="finished">
+                <h2>You just migrated:</h2>
+                <ul>
+                    <li>
+                        <i class="fa-brands fa-bluesky bluesky-bullet" />
+                        <strong>{{ model.progress.migrateTweetsCount.toLocaleString() }}</strong> tweets to
+                        Bluesky
+                    </li>
+                </ul>
+
+                <div v-if="model.progress.migrateSkippedTweetsCount > 0" class="alert alert-warning mt-4">
+                    <p>
+                        <strong>
+                            {{ model.progress.migrateSkippedTweetsCount.toLocaleString() }} tweets
```

Comment:
nit: this will result in "1 tweets were skipped" (should have singular and plural cases, could punt to followup issue)